addons/l10n_pt/models/account_journal.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    restrict_mode_hash_table = fields.Boolean(compute='_compute_restrict_mode_hash_table')
```

Comment:
Declare the field as computed in `account`It is really weird and most likely wrong to change it from related to computed like this